### ISSUE TITLE
Compiling error using vue-cli

### ISSUE THREAD

**Issue by mrleblanc101** [alvarotrigo/vue-fullpage.js#17]
Error:
```
Add @babel/plugin-proposal-object-rest-spread (https://git.io/vb4Ss) to the 'plugins' section of your Babel config to enable transformation.
```

My .babelrc: 
```json
{
  "presets": [
    "@vue/app"
  ],
  "plugins": [
    "transform-object-rest-spread",
    "@babel/plugin-proposal-object-rest-spread"
  ]
}
```

Build trace: 
```
Trace: The node type SpreadProperty has been renamed to SpreadElement
    at Object.isSpreadProperty (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/types/lib/validators/generated/index.js:4080:11)
    at hasSpread (/Users/mrleblanc/GitHub/portfolio/node_modules/babel-plugin-transform-object-rest-spread/lib/index.js:38:13)
    at PluginPass.ObjectExpression (/Users/mrleblanc/GitHub/portfolio/node_modules/babel-plugin-transform-object-rest-spread/lib/index.js:234:14)
    at newFn (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/visitors.js:243:21)
    at NodePath._call (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/path/context.js:65:18)
    at NodePath.call (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/path/context.js:40:17)
    at NodePath.visit (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/path/context.js:100:12)
    at TraversalContext.visitQueue (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/context.js:144:16)
    at TraversalContext.visitMultiple (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/context.js:99:17)
    at TraversalContext.visit (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/context.js:183:19)
    at Function.traverse.node (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/index.js:106:17)
    at NodePath.visit (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/path/context.js:107:18)
    at TraversalContext.visitQueue (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/context.js:144:16)
    at TraversalContext.visitSingle (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/context.js:104:19)
    at TraversalContext.visit (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/context.js:185:19)
    at Function.traverse.node (/Users/mrleblanc/GitHub/portfolio/node_modules/@babel/traverse/lib/index.js:106:17)
```

**Comment by ismet** [comment]
I got exactly the same error as @mrleblanc101
I could fix it by doing what he suggests: "run npm install in /node_modules/vue-fullpage.js"
The issue must be related with vue-fullpage.js.

**Comment by alvarotrigo** [comment]
This should be now fixed on the latest version 0.0.2
